spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -154,9 +154,9 @@
   scenario "User publishes changes to a published step by step page" do
     given_there_is_a_published_step_by_step_page_with_unpublished_changes
     when_I_view_the_step_by_step_page
-    and_when_I_click_button "Publish changes"
+    and_when_I_click_button "Publish"
     then_I_should_see_a_publish_form_with_changenotes
-    and_when_I_click_button "Publish step by step"
+    and_when_I_click_the_publish_button_in_the_publish_form "Publish"
```

Comment:
Hmm. This reads strangely. I wasn't expecting `and_when_I_click_the_publish_button_in_the_publish_form` to take a parameter.

Reply:
Right.. I removed the alias and created a separate function for it that doesn't require the button label.